### PR TITLE
Broadcast error event `ContactIdNotFound`

### PR DESCRIPTION
**Pull request by charliewang95**
Parity feature PR for Enhanced Auth 

image

### CODE REVIEW

**Review by elopezanaya — APPROVED** [review]

File: chat-widget/package.json [hunk 1]
```diff
@@ -74,9 +74,8 @@
     "webpack-cli": "^4.9.2"
   },
   "dependencies": {
-    "@fluentui/react": "^8.49.1",
     "@microsoft/omnichannel-chat-components": "1.0.1",
-    "@microsoft/omnichannel-chat-sdk": "1.3.0",
+    "@microsoft/omnichannel-chat-sdk": "^1.3.1-main.7f52e0c",
```
Comment (akhiloct1510) [inline chat-widget/package.json:78]: Is this intentional ?

File: chat-widget/src/common/utils.ts [hunk 2]
```diff
@@ -386,3 +393,16 @@ export const getConversationDetailsCall = async (chatSDK: any) => {
 
     return conversationDetails;
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const checkContactIdError = (e: any) => {
+    if (e?.message === "AuthContactIdNotFoundFailure") {
```
Comment (akhiloct1510) [inline chat-widget/src/common/utils.ts:399]: Can we declare this as constants and use it from there ?

**Review by akhiloct1510 — APPROVED** [review]

File: chat-widget/package.json (hunk 1 above)
Comment (charliewang95) [reply chat-widget/package.json:78]: Will change to formal version once it's released

File: chat-widget/src/common/utils.ts (hunk 2 above)
Comment (xTEddie) [reply chat-widget/src/common/utils.ts:399]: It's a `ChatSDKError`. Shall we create a `Enum` for this?

**Pull request merged**